src/main.zig

```diff
@@ -133,27 +133,27 @@ pub const Style = struct {
         self.is_empty = self.is_empty and other.is_empty;
     }
 
-    pub fn set_underline(self: *Self, underline: bool) void {
+    pub fn setUnderline(self: *Self, underline: bool) void {
```

Comment:
I'm surprised zls doesn't warn about these despite the "code style" option being turned on :thinking: